test/token_utils_test.dart

```diff
@@ -0,0 +1,479 @@
+      });
+    });
+
+    group('getOrganization', () {
+      test('should return organization code', () {
+        tokenUtils.authState = AuthStateFixtures.createWithOrganization('org_acme');
+
+        final org = tokenUtils.getOrganization();
+
+        expect(org.orgCode, 'org_acme');
+      });
+
+      test('should return org code from token', () {
+        tokenUtils.authState = AuthStateFixtures.createWithOrganization('my_org');
+
+        final org = tokenUtils.getOrganization();
+
+        expect(org.orgCode, 'my_org');
+      });
+
+      test('should handle null auth state', () {
+        tokenUtils.authState = null;
+
+        final org = tokenUtils.getOrganization();
+
+        expect(org.orgCode, '');
+      });
+    });
+
+    group('getUserOrganizations', () {
+      test('should return all organization codes from id token', () {
+        tokenUtils.authState = AuthStateFixtures.createValid();
+
+        final orgs = tokenUtils.getUserOrganizations();
+
+        expect(orgs.orgCodes, hasLength(2));
+        expect(orgs.orgCodes, contains('org_test123'));
+        expect(orgs.orgCodes, contains('org_other456'));
+      });
+
+      test('should return empty list when org_codes excluded', () {
+        final idToken = JwtTestHelper.createIdToken(excludeOrgCodes: true);
+        tokenUtils.authState = AuthState(
+          idToken: idToken,
+          accessToken: JwtTestHelper.createAccessToken(),
+        );
+
+        final orgs = tokenUtils.getUserOrganizations();
+
+        expect(orgs.orgCodes, isEmpty);
+      });
+
+      test('should handle null auth state', () {
+        tokenUtils.authState = null;
+
+        final orgs = tokenUtils.getUserOrganizations();
+
+        expect(orgs.orgCodes, isEmpty);
+      });
+    });
+
+    group('getFlag', () {
+      test('should return boolean flag', () {
+        tokenUtils.authState = AuthStateFixtures.createWithFeatureFlags({
+          'is_dark_mode': {'t': 'b', 'v': true},
+        });
+
+        final flag = tokenUtils.getFlag(code: 'is_dark_mode');
+
+        expect(flag, isNotNull);
+        expect(flag!.code, 'is_dark_mode');
+        expect(flag.type, FlagType.bool);
+        expect(flag.value, true);
+        expect(flag.isDefault, false);
+      });
+
+      test('should return string flag', () {
+        tokenUtils.authState = AuthStateFixtures.createWithFeatureFlags({
+          'theme_color': {'t': 's', 'v': 'blue'},
+        });
+
+        final flag = tokenUtils.getFlag(code: 'theme_color');
+
+        expect(flag, isNotNull);
+        expect(flag!.type, FlagType.string);
+        expect(flag.value, 'blue');
+      });
+
+      test('should return integer flag', () {
+        tokenUtils.authState = AuthStateFixtures.createWithFeatureFlags({
+          'max_items': {'t': 'i', 'v': 100},
+        });
+
+        final flag = tokenUtils.getFlag(code: 'max_items');
+
+        expect(flag, isNotNull);
+        expect(flag!.type, FlagType.integer);
```

Comment:
```suggestion
        expect(flag!.code, 'max_items');
        expect(flag.type, FlagType.integer);
```

Checking the code to match the boolean flag assertion.